### PR TITLE
[MIA] [Gyasik - Nunez]

### PR DESCRIPTION
**Pull request by gyasikn**
Issues: 

- Resizing the apple pie & french soup photos. I couldn't get them to scale without overflowing or squishing them, so I placed them in the size that I wanted with a scroller for now so they're actually there.

### CODE REVIEW

File: starter-code/apple-pie-starter-code/css/main.css [hunk 1]
```diff
@@ -6,34 +6,53 @@
...
-    background-image: url("images/apple-pie.jpg");
-    background-position: 0 0;
-    background-repeat: no-repeat;
-    background-size: cover;
-    height: 650px;
+.cover-photo {
+    width:80%;
+    height: 500px;
+    margin: auto;
+    overflow: scroll;
```
Comment (sandrabosk) [inline starter-code/apple-pie-starter-code/css/main.css]:
```
overflow: hidden;
```
And that will fix your issue with scroll bar.

File: starter-code/french-onion-soup-starter-code/css/main.css [hunk 2]
```diff
@@ -1,9 +1,69 @@
...
+    left: 22%;
+    color: white;
+    font-weight: 300;
+}
+
+.cover-photo {
+    width: 80%;
+    height: 500px;
+    margin: auto;
+    overflow: scroll;
```
Comment (sandrabosk) [inline starter-code/french-onion-soup-starter-code/css/main.css]:
```
overflow: hidden;
```

**Comment by sandrabosk** [comment]
Very good job!

File: starter-code/apple-pie-starter-code/css/main.css (hunk 1 above)
Comment (gyasikn) [reply starter-code/apple-pie-starter-code/css/main.css]: I had overflow:hidden before but it still gives issues. check the new pull request I just sent with the changes.

Comment (sandrabosk) [reply starter-code/apple-pie-starter-code/css/main.css]: I don't see any issues when using hidden. What is the issue you're dealing with?

**Pull request closed without merging**